sklearn/mixture/_base.py

```diff
@@ -136,8 +136,10 @@ def _initialize_parameters(self, X, random_state):
         ----------
         X : array-like, shape  (n_samples, n_features)
 
-        random_state : RandomState
-            A random number generator instance.
+        random_state : int, RandomState, default=None
+            A random number generator instance. Pass an
```

Comment:
This should be reverted. For this private method, the random_state can only be a random state object.